applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -147,6 +147,13 @@ Check and configure the following configuration options for the sample:
 
    This option is selected by default.
 
+.. option:: CONFIG_SLM_CONFIRM_XSLEEP - Send response on sleep command
+
+   This option makes nRF9160 enter deep sleep after startup.
```

Comment:
Maybe we need to mention that this option enables the OK response. Otherwise, there is no difference between this and CONFIG_SLM_START_SLEEP

Reply:
Thanks, this was a forgotten update, fixed.